lua/luau-lsp/config.lua

```diff
@@ -30,6 +30,10 @@ local defaults = {
     ---@type table<string, "True"|"False"|number>
     override = {},
   },
+  companion = {
```

Comment:
The setting name should be "plugin" instead of "companion" and should be disabled by default.